frontend/src/utils/auth.js

```diff
@@ -12,6 +12,8 @@ export function parseToken(token) {
 
   const data = JSON.parse(Base64.decode(parts[1]));
 
+  document.cookie = `auth=${token}; path=/`;
```

Comment:
I don't think that it is a good idea to create an auth cookie accessible by clients. I would suggest generating an anonymous JWT token for unauthorized users. As an option, we can create a special file access token instead of using auth token for that.

Reply:
The token is already acessible by the client by being stored on localStorage. What makes the cookie approach inferior to the one that we already have? The objetive of the cookie is authenticate browser requests without the auth query param, which is required to make the cache work.